### ISSUE TITLE
multi git config

### ISSUE THREAD

**Issue by improve100** [kahole/vscode-magit#184]
if i have multi git repo in the folder, can i config magit use which one?
like this, when i open a.txt i use A repo.otherwise use B
root
   A
     .git
      a.txt
   B
    .git
    b.txt

**Comment by kahole** [comment]
In that case edamagit should open the repo closest to the file in question. Can you describe what you expect in some more detail, or lay out the steps to reproduce the situation?

**Comment by improve100** [comment]
Yes, just like magit in Emacs will look for whether there is .git in the current  folder, if had, use the current GIT, otherwise would continue to find the previous directory.

**Comment by kahole** [comment]
That is how it works. Please add steps to reproduce your issue

**Comment by improve100** [comment]
vscode: Version: 1.61.2
edamagit: v0.6.23
os: ubuntu 1604
directory structure：
proj
.... .git
.... install
........ 1.txt
........ 2.txt
.... src
........ .git
........ 10.txt
........ 20.txt
.... others
........ .git
........ 100.txt
........ 200.txt

when i open 1.txt,10.txt,100.txt then alt+x g it show:
![image](https://user-images.githubusercontent.com/6092914/139017877-c0760178-4a2a-49bd-85a1-1056c448e138.png)
![image](https://user-images.githubusercontent.com/6092914/139017998-47dd6f8a-2b3e-4236-97d4-987d4476d3f8.png)
![image](https://user-images.githubusercontent.com/6092914/139017749-7bb9e38c-9439-47a0-a252-c9c04fc9a223.png)

test folder:
[proj.zip](https://github.com/kahole/edamagit/files/7424021/proj.zip)
